File: objc/objc/ViewController.m
//
//  ViewController.m
//  objc
//
//  Created by Dev on 17/04/2024.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}


- (IBAction)buttonAction:(UIButton *)sender {
    //NSLog(@"hi");
    self.label.text = @"Welcome";
}
@end

File: NavigationWithoutNavController/NavigationWithoutNavController/ViewController.swift
//
//  ViewController.swift
//  NavigationWithoutNavController
//
//  Created by Dev on 23/10/2023.
//

import UIKit

class ViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    
    @IBAction func GoAhead(_ sender: UIButton) {
        let secondViewController = storyboard?.instantiateViewController(withIdentifier: "SecondViewController") as! SecondViewController
            // Push the new view controller onto the navigation stack
        
            navigationController?.pushViewController(secondViewController, animated: true)
    }
    
}
